prow/config.yaml

```diff
@@ -526,3 +526,34 @@ presubmits:
       - name: test-account
         secret:
           secretName: test-account
+
+postsubmits:
+  tektoncd/dashboard:
+  - name: tekton-dashboard-publish-images
+    always_run: true
+    spec:
+      containers:
+      - image: gcr.io/tekton-releases/tests/test-runner@sha256:a4a64b2b70f85a618bbbcc6c0b713b313b2e410504dee24c9f90ec6fe3ebf63f
+        imagePullPolicy: Always
+        args:
+        - "--scenario=kubernetes_execute_bazel"
+        - "--clean"
+        - "--job=$(JOB_NAME)"
+        - "--repo=github.com/$(REPO_OWNER)/$(REPO_NAME)=$(PULL_REFS)"
+        - "--root=/go/src"
+        - "--service-account=/etc/nightly-account/service-account.json"
+        - "--upload=gs://tekton-prow/pr-logs"
+        - "--" # end bootstrap args, scenario args below
+        - "--" # end kubernetes_execute_bazel flags (consider following flags as text)
+        - "./test/postsubmit-tests.sh"
```

Comment:
It looks like this hasn't been merged yet, maybe a better name would be "publish_imagesh.sh" instead of postsubmit-tests? Unless we're also planning to add tests.

Reply:
@dlorenc good point, I think it's fine naming it `publish_image.sh` :wink: I'll update the dashboard PR too :wink:

Reply:
@dlorenc `dashboard` PR updated :wink: